package.json

```diff
@@ -19,8 +19,8 @@
     "chai-moment": "^0.1.0",
     "coveralls": "^2.11.12",
     "eslint": "^3.2.2",
-    "eslint-config-airbnb": "^10.0.0",
-    "eslint-plugin-import": "^1.8.1",
+    "eslint-config-airbnb": "^13.0.0",
+    "eslint-plugin-import": "^2.1.0",
```

Comment:
Because we use ^ both will have 2.2 installed in a clean install.

There is probably a discussion to be had around the usage of ^ vs ~ vs "